resources/views/mail/notification/set-revision.blade.php

```diff
@@ -5,9 +5,11 @@
 <x-mail::message>
 Hello {{ $user->display_name }},
 
-An achievement set you have previously {{ $isHardcore ? 'mastered' : 'completed' }} has been revised.
+The achievement set for [{{ $game->title }}]({{ $gameUrl }}) has been updated.
 
-Check out the changes to [{{ $game->title }}]({{ $gameUrl }}).
+**Your {{ $isHardcore ? 'mastery' : 'completion' }} status is safe.** You've already earned your badge, and it isn't going anywhere.
+
+If you're curious about what changed or want an excuse to revisit this retro game, [check out its game page]({{ $gameUrl }}). You've already {{ $isHardcore ? 'mastered' : 'completed' }} it - this may be a great chance to earn even more points.
```

Comment:
There are two links to the game page in this email. I'm not sure if that's a bad thing.